providers/microsoft/azure/tests/conftest.py

```diff
@@ -89,109 +69,10 @@ def wrapper(*conns: T):
 
 @pytest.fixture
 def mocked_connection(request, create_mock_connection):
-    """Helper indirect fixture for create test connection."""
+    """Create test connection."""
     return create_mock_connection(request.param)
 
 
-def mock_connection(schema: str | None = None, host: str | None = None):
```

Comment:
Moved all those from conftest to `test_utils`. Those functions are imported in tests and importing stuff from conftest.py is not going to work as you would just impot "from conftest" - this is also not the intention of conftest.py -> conftest.py is automatically read by pytest and fixtures are configured and made available from it - it was never supposed to be used to import things from.